labcontrol/gui/handlers/process_handlers/test/test_pooling_process.py

```diff
@@ -65,25 +65,74 @@ def test_post_pool_pool_process_handler(self):
+        response = self.get('/process/poollibraries/amplicon_sequencing/'
+                            '?plate_id=23')
         self.assertEqual(response.code, 200)
         self.assertNotEqual(response.body, '')
 
-        response = self.get('/process/poollibraries?plate_id=23')
+        # plate_id=23 is an amplicon plate, so this should process normally
+        response = self.get('/process/poollibraries/amplicon_sequencing/'
+                            '?plate_id=26')
+        self.assertEqual(response.code, 400)
+        self.assertRegexpMatches(response.body, rb'Plate type does not match')
+
+        # plate_id=26 is a shotgun plate, so this should process normally
+        response = self.get('/process/poollibraries/shotgun_plate/'
+                            '?plate_id=26')
         self.assertEqual(response.code, 200)
         self.assertNotEqual(response.body, '')
 
-        response = self.get('/process/poollibraries?plate_id=23&plate_id=26')
+        # should get a 400 error from having plates of different type
+        response = self.get('/process/poollibraries/amplicon_sequencing/'
+                            '?plate_id=23&plate_id=26')
+        self.assertEqual(response.code, 400)
+        self.assertRegexpMatches(response.body, rb'Plates contain different')
+        response = self.get('/process/poollibraries/shotugn_plate/'
+                            '?plate_id=23&plate_id=26')
         self.assertEqual(response.code, 400)
+        self.assertRegexpMatches(response.body, rb'Plates contain different')
 
-        response = self.get('/process/poollibraries?process_id=1')
+        # process_id=1 corresponds to an amplicon plate, so this retrieve
+        # the info
+        response = self.get('/process/poollibraries/amplicon_sequencing/'
+                            '?process_id=1')
         self.assertEqual(response.code, 200)
         self.assertNotEqual(response.body, '')
 
-        response = self.get('/process/poollibraries?process_id=10000')
+        # process_id=1 corresponds to an amplicon plate, so this should
+        # raise an error
+        response = self.get('/process/poollibraries/shotgun_plate/'
+                            '?process_id=1')
+        self.assertEqual(response.code, 400)
+        self.assertRegexpMatches(response.body, rb'process type does not '
+                                                rb'match')
+
+        # this process id does not exist
+        response = self.get('/process/poollibraries/amplicon_sequencing/'
+                            '?process_id=10000')
+        self.assertEqual(response.code, 404)
+
+        # this process id does not exist
+        response = self.get('/process/poollibraries/shotgun_plate/'
+                            '?process_id=10000')
         self.assertEqual(response.code, 404)
 
     def test_post_library_pool_process_handler(self):
 
+        # TODO may want to include test to make sure it is not able to post
+        #  plates with the wrong type
```

Comment:
this is the most minor issue possible but there's an extra space before "plates"

Reply:
I usually put a second space in continuation lines of comments. I like this convention, but feel free to change.